test/e2e/flowaggregator_test.go

```diff
@@ -754,10 +790,21 @@ func getUnit64FieldFromRecord(t *testing.T, record string, field string) uint64
 // received all the expected records for a given flow with source IP, destination IP
 // and source port. We send source port to ignore the control flows during the
 // iperf test.
-func getCollectorOutput(t *testing.T, srcIP, dstIP, srcPort string, timeStart time.Time, checkAllRecords bool) (string, []string) {
+func getCollectorOutput(t *testing.T, srcIP, dstIP, srcPort string, timeStart time.Time, checkAllRecords bool, restartInfo restartInfo) (string, []string) {
 	var collectorOutput string
 	var recordSlices []string
-	err := wait.PollImmediate(500*time.Millisecond, aggregatorInactiveFlowRecordTimeout, func() (bool, error) {
+	timeout := aggregatorInactiveFlowRecordTimeout
+	if restartInfo.isRestart {
+		timeout = restartFlowAggregatorTimeout
+	}
+	err := wait.PollImmediate(500*time.Millisecond, timeout, func() (bool, error) {
+		// When checking restart flows, we need to create traffic until one flow record
+		// is received correctly since connection re-establish between Flow Exporter and
+		// Flow Aggregator will happen any time during connCheckInterval.
```

Comment:
"For the restartFlowAggregator test, we create traffic in every poll as we do not know exactly when the connection is re-established from the Flow Exporter to the Flow Aggregator. We will modify the test when there is visibility into connection re-establishment in the future."